.github/workflows/full_stack_tests.yml

```diff
@@ -155,7 +158,24 @@ jobs:
           SUITE_NAME=${{ matrix.suite.name }}
           echo "MODIFIED_SUITE_NAME=${SUITE_NAME//\//_}" >> $GITHUB_OUTPUT
       - name: Run Desktop Acceptance Test ${{ matrix.suite.name }}
-        run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_acceptance_tests --skip-build --suite=${{ matrix.suite.name }} --prod_env
+        run: >
+          VIDEO_RECORDING_IS_ENABLED=0
+          xvfb-run -a --server-args="-screen 0, 1285x1000x24" 
+          python -m scripts.run_acceptance_tests 
+          --skip-build --suite=${{ matrix.suite.name }} --prod_env
+      - name: Run Desktop Acceptance Test ${{ matrix.suite.name }} (Attempt 2 with screen recording)
```

Comment:
If this second run passes, the workflow should still fail. Does it?

Reply:
Yes, verified through following job: https://github.com/oppia/oppia/actions/runs/12971427846/job/36177997710?pr=21775

Reply:
@seanlip It will show full log as it used to be before. In the above workflow it's not showing because job actually passed. I forcefully exited with code 1 to show that even if Attempt 2 passes, workflow fails.

You can check full error code in job (mentioned in PR description) where job fails: https://github.com/oppia/oppia/actions/runs/12921861547/job/36036898504?pr=21775. However, it doesn't give proof to your following question. So, I forcefully failed attempt 1.
> If this second run passes, the workflow should still fail. Does it?

Reply:
Oh, I see. Yeah, I think that's OK then -- thanks!